python/setup.py

```diff
@@ -0,0 +1,8 @@
+from setuptools import setup
+
+setup(
+    name='lolopy',
+    version='0.0.1',
+    packages=['lolopy'],
+    description='Python wrapper for the Lolo machine learning library',
```

Comment:
We should add sklearn and py4j to install_requirements.

Reply:
Done